tiledb/common/status_code.h

```diff
@@ -95,7 +95,10 @@ enum class StatusCode : char {
   FragmentInfoError,
   DenseTilerError,
   QueryConditionError,
-  ArrayDirectoryError
+  ArrayDirectoryError,
+  Group,
+  GroupDirectoryError,
+  GroupMemberError
 };
 
 std::string to_string(const StatusCode& sc);
```

Comment:
Not directly related, but there's an implicit `include <string_view>` here which should be made explicit, if you don't mind.